pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -598,7 +597,7 @@ func (p *PriorityQueue) Pop() (*framework.QueuedPodInfo, error) {
 		// When Close() is called, the p.closed is set and the condition is broadcast,
 		// which causes this loop to continue and return from the Pop().
 		if p.closed {
-			return nil, fmt.Errorf(queueClosed)
```

Comment:
Still prefer to return error in schedulingQueue to store the original reason, but in scheduler, since the error is because of the termination of scheduler, let's log this with info level rather than `klog.Errors`.